### DISCUSSION TITLE
on windows resize (browser windows Zoom In or zoom out) object are going outside the controlls box

### DISCUSSION THREAD

**Discussion by mostafasalloumm** [fabricjs/fabric.js#7970] [General]
![Screenshot 2022-05-25 223447](https://user-images.githubusercontent.com/81439418/170353608-214a5f93-d11b-4bed-8144-30c893890b21.png)

**Comment by xorb** [comment]
Could you share a fiddle  reproducing the issue?

**Reply by ShaMan123** [reply]
seems like a setCoords/calcCanvasOffset issue
